src/amo/components/UserProfileEdit/index.js

```diff
@@ -440,7 +441,7 @@ export class UserProfileEditBase extends React.Component<Props, State> {
               className="UserProfileEdit--Card"
               header={i18n.gettext('Profile')}
             >
-              <p className="UserProfileEdit-aside">
+              <p className="UserProfileEdit-profile-aside">
```

Comment:
Both of these new classes, `UserProfileEdit-profile-aside` and `UserProfileEdit-notifications-aside`, point to the same css definition (which used to be just `UserProfileEdit-aside`), so why create two different classes for them? Why not just add the existing `UserProfileEdit-aside` class to the new aside you created for notifications?

Reply:
I want to be able to use them in the test suite (I think).